index.yaml

```diff
@@ -6084,30 +6084,6 @@ entries:
     urls:
     - assets/prometheus-federator/prometheus-federator-104.0.0-rc1+up0.4.1.tgz
     version: 104.0.0-rc1+up0.4.1
-  - annotations:
```

Comment:
Did you mean to delete this? I don't see how this relates to this PR.

Reply:
Hmm, no I didn't it looks like this was done automatically when using the charts build scripts. Let me see if I can avoid this from happening

Reply:
Okay, most recent pushes have resolved this issue

Reply:
@adamkpickering  good catch!